rpm/zstd.spec

```diff
@@ -0,0 +1,111 @@
+%ifarch %{ix86} x86_64
+%bcond_without pzstd
+%else
+# aarch64 and armv7hl at least currently segfault
+# in ThreadPool test for the pzstd util
+%bcond_with pzstd
+%endif
+
+Name:           zstd
+Version:        1.5.0
+Release:        1
+Summary:        Zstd compression library
+
+License:        BSD and GPLv2
+URL:            https://github.com/facebook/zstd
+Source0:        https://github.com/facebook/zstd/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:         0001-Use-r-for-extended-regex-instead-of-E-as-our-sed-is-.patch
+
+BuildRequires: make
+BuildRequires:  gcc gtest-devel
+%if %{with pzstd}
+BuildRequires:  gcc-c++
+%endif
+
+%description
+Zstd, short for Zstandard, is a fast lossless compression algorithm,
+targeting real-time compression scenarios at zlib-level compression ratio.
+
+%package -n lib%{name}
+Summary:        Zstd shared library
+
+%description -n lib%{name}
+Zstandard compression shared library.
+
+%package -n lib%{name}-devel
+Summary:        Header files for Zstd library
+Requires:       lib%{name}%{?_isa} = %{version}-%{release}
+
+%package -n lib%{name}-static
+Summary:        Static variant of the Zstd library
+Requires:       lib%{name}-devel = %{version}-%{release}
+
+%description -n lib%{name}-devel
+Header files for Zstd library.
+
+%description -n lib%{name}-static
+Static variant of the Zstd library.
+
+%prep
+%autosetup -n %{name}-%{version}/%{name} -p1
+find -name .gitignore -delete
+
+%build
+export CFLAGS="$RPM_OPT_FLAGS"
+export LDFLAGS="$RPM_LD_FLAGS"
+%make_build -C lib lib-mt
+%make_build -C programs
+%if %{with pzstd}
+export CXXFLAGS="$RPM_OPT_FLAGS"
+%make_build -C contrib/pzstd
+%endif
+
+%check
+export CFLAGS="$RPM_OPT_FLAGS"
+export LDFLAGS="$RPM_LD_FLAGS"
+# disabled tests as they require bigger workers and take quite a bit
```

Comment:
If we want to keep the tests deactivated in the spec we should convert them into a build condition.

Reply:
Added that